packages/web/test.js

```diff
@@ -9,7 +9,8 @@ const playwright = require('playwright')
 const fs = require("fs")
 const handler = require("serve-handler")
 const http = require("http")
-const path = require("path")
+const path = require("path");
```

Comment:
the semi colon in both is actually not needed

Reply:
Fixed at [91aa76e](https://github.com/thingweb/thingweb-playground/pull/439/commits/91aa76e75dab77f00023fcd5a386a40eb1ad2d67)